ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java

```diff
@@ -153,57 +157,24 @@ private void processInputUri() throws NullPointerException, IOException {
                 Log.e(TAG, "Downloading failed", e);
                 throw e;
             }
-        } else if ("content".equals(inputUriScheme)) {
```

Comment:
Why file copying for contente scheme was removed?

Reply:
It's been so long since I made there changes that I don't really remember all details. This PR is from January. I've worked on this code on the and of 2020! In fact this project seemed to be abandoned and I was even thinking about following with a forked project, I just haven't had the time to do that yet (see the conversations on the issue #715 and see my comment here: https://github.com/Yalantis/uCrop/pull/732#issuecomment-766408792).

But in this particular commit my comment was:

> Fix for problems with input and output File Uri with "content" schema.
> Fix for using uCrop with SAF (Storage Access Framework) on Android 10 and newer.
> Sample app changed for context file testing. Added a "choose file destination" button and some other associated controls.

I don't remember exactly why but I do remember that this part of the code was avoiding me to make the code work for SAF and with the modifications I've made it became obsolete as I was dealing with **content** scheme in a broader way. I remember searching in the log files the reason this copy was been done because the method was copying the file from the input uri to the output uri and then pointing the input uri to the same object as the output uri:

```java
 private void copyFile(@NonNull Uri inputUri, @Nullable Uri outputUri) throws NullPointerException, IOException {
        Log.d(TAG, "copyFile");

        if (outputUri == null) {
            throw new NullPointerException("Output Uri is null - cannot copy image");
        }

        InputStream inputStream = null;
        OutputStream outputStream = null;
        try {
            inputStream = mContext.getContentResolver().openInputStream(inputUri);
            outputStream = new FileOutputStream(new File(outputUri.getPath()));
            if (inputStream == null) {
                throw new NullPointerException("InputStream for given input Uri is null");
            }

            byte buffer[] = new byte[1024];
            int length;
            while ((length = inputStream.read(buffer)) > 0) {
                outputStream.write(buffer, 0, length);
            }
        } finally {
            BitmapLoadUtils.close(outputStream);
            BitmapLoadUtils.close(inputStream);

            // swap uris, because input image was copied to the output destination
            // (cropped image will override it later)
            mInputUri = mOutputUri;
        }
    }
```

And latter if cropped in the `BitmapCropTask.saveImage` it was been written again and if not cropped it was been copied again at the end of `BitmapCropTask.crop`. 
```java
FileUtils.copyFile(mImageInputPath, mImageOutputPath);
```
This copy was not been useful, was duplicated, or became obsolete with my modifications. I can't point the most exact reason now since it's been half a year since then.
The main point of this PR is to allow using SAF with uCrop booth with the source URI and the destination URI and when there is a "content" schema is exactly when we are dealing with SAF. If I simply override the destination or origin URI I loose the permissions attached to it.

Reply:
Since *content* schema now is been handled on `BitmapCropTask.crop` this particular copying is not util anymore and so it was deleted. Do you agree with this change?

Reply:
Yeah, got it, thanks for explanation